nuxeo-opencmis-impl/src/main/java/org/nuxeo/ecm/core/opencmis/impl/server/CMISQLtoNXQL.java

```diff
@@ -301,6 +303,12 @@ public IterableQueryResult getIterableQueryResult(IterableQueryResult it, NuxeoC
         return new NXQLtoCMISIterableQueryResult(it, realColumns, virtualColumns, service);
     }
 
+    public PartialList<Map<String, Serializable>> getPartialList(PartialList<Map<String, Serializable>> pl,
```

Comment:
I'm not sure `getPartialList` is the best name, as it both takes and returns a `PartialList` already. What it does is a form of conversion.